app/javascript/hello_world.test.js

```diff
@@ -1,5 +0,0 @@
-describe('hello world', () => {
```

Comment:
I deleted this file because this test is no longer necessary - I added a real test with this PR.